ArduPlane/quadplane.cpp

```diff
@@ -730,10 +738,10 @@ bool QuadPlane::setup(void)
     }
     AP_Param::load_object_from_eeprom(loiter_nav, loiter_nav->var_info);
 
-    motors->init((AP_Motors::motor_frame_class)frame_class.get(), (AP_Motors::motor_frame_type)frame_type.get());
+    motors->initialise(motor_class, motor_type);
```

Comment:
Why the name change here?  `init(...)` is pretty traditional.

Reply:
gone